react-app/src/library/services/vendor-implementations/jabra/jabra.ts

```diff
@@ -370,16 +372,29 @@ export default class JabraService extends VendorImplementation {
     this.changeConnectionStatus({ isConnected: true, isConnecting: false });
   }
 
+  async deviceHasPermissions (deviceLabel: string): Promise<boolean> {
```

Comment:
Originally we were pulling devices from Jabra by using an Observable.  Unfortunately, we didn't have a way to determine if there were no devices so we had a 3 second time out which caused issues down the road.  So this new approach pulls the devices from the HID API (which is much quicker) and checks to see if the provided device label exists in the returned list.  If it does not, then we pop the modal.  But if it does exist, then we pass it over to the original Jabra Observable to pull the proper device with all the necessary pieces that Jabra requires.  This should then circumvent the need for the timer.